### ISSUE TITLE
NVDA doesn't read code window of Dev c++

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1223]
**Reported by oaron on 2010-11-26 20:10**
When I create or open a cpp file with Dev c++, and press up and down arrows, NVDA doesn't announce the lines. But When I move the review cursor, NVDA announce it.
Can we do something with this problem?

**Comment by leonardder** [comment]
@oaron: Could you provide additional details about this?

**Comment by leonardder** [comment]
This ticket looks quite abandoned and we seem nunable to contact the author. Closing.
